Scheduler/feature_scheduler/maintel/fbs_config_sit_survey_block_t189.py

```diff
@@ -0,0 +1,194 @@
+    # think these are fairly set, so no need to promote to utility func kwargs
+    greed_survey_params = {
+        "block_size": 1,
+        "smoothing_kernel": None,
+        "seed": seed,
+        "camera": "LSST",
+        "dither": True,
+        "survey_name": "BLOCK-T87",
+    }
+
+    surveys = []
+    detailer = detailers.CameraRotDetailer(
+        min_rot=np.min(camera_rot_limits), max_rot=np.max(camera_rot_limits)
+    )
+
+    exptimes = {f: exptime for f in filters}
+    nexps = {f: nexp for f in filters}
+    if isinstance(exptime_override, dict):
+        exptimes.update(exptime_override)
+    if isinstance(nexp_override, dict):
+        nexps.update(nexp_override)
+
+    for filtername in filters:
+        bfs = [
+            (
+                bf.FootprintBasisFunction(
+                    filtername=filtername,
+                    footprint=footprints,
+                    out_of_bounds_val=np.nan,
+                    nside=nside,
+                ),
+                footprint_weight,
+            ),
+            (
+                bf.SlewtimeBasisFunction(filtername=filtername, nside=nside),
+                slewtime_weight,
+            ),
+            (bf.StrictFilterBasisFunction(filtername=filtername), stayfilter_weight),
+            (
+                bf.ZenithShadowMaskBasisFunction(
+                    nside=nside,
+                    shadow_minutes=shadow_minutes,
+                    max_alt=max_alt,
+                    min_alt=30.0,
+                ),
+                0,
+            ),
+            (bf.FilterLoadedBasisFunction(filternames=filtername), 0),
+        ]
+
+        weights = [val[1] for val in bfs]
+        basis_functions = [val[0] for val in bfs]
+        surveys.append(
+            GreedySurvey(
+                basis_functions,
+                weights,
+                exptime=exptimes[filtername],
+                filtername=filtername,
+                nside=nside,
+                ignore_obs=ignore_obs,
+                nexp=nexps[filtername],
+                detailers=[detailer],
+                **greed_survey_params,
+            )
+        )
+
+    return surveys
+
+
+if __name__ == "config":
+    nside = 32
+    per_night = True  # Dither DDF per night
+    seed = 42
+
+    camera_ddf_rot_limit = 75.0
+
+    observatory = ModelObservatory(nside=nside, mjd_start=MJD_START)
+    observatory.sky_model.load_length = 3
+    conditions = observatory.return_conditions()
+
+    sky = SkyAreaGenerator(nside=nside)
+    footprints_hp, footprints_labels = sky.return_maps()
+
+    footprints = Footprint(
+        conditions.mjd_start, sun_ra_start=conditions.sun_ra_start, nside=nside
+    )
+    for i, key in enumerate(footprints_hp.dtype.names):
+        footprints.footprints[i, :] = footprints_hp[key]
+
+    # Generate surveys for all filters to test "FilterLoaded" basis func
+    eo_test_filters = ["u", "y", "g", "i", "r", "z"]  # ['y', 'r', 'g'] actually present
```

Comment:
How are you expecting the filters to be selected by the Scheduler?

Reply:
This is where my TTS testing was not able to help me. I am not sure if this is functionality actually exists. I am hoping I that the scheduler's observatory model can ingest `MTCamera_logevent_availableFilters`. I could not confirm this piping exists. I may also need to explicitly subscribe to this in `maintel_fbs_sit_block_t189.yaml`, please advise.

Reply:
Unfortunately, that mechanism does not exist yet. But you can control this by updating this configuration parameter: https://github.com/lsst-ts/ts_config_ocs/blob/31ce77d4259bfb4c3e6d0567706af8ee48d61db3/Scheduler/v7/_init.yaml#L174-L179

Reply:
I am not sure if it works with less than 5 filters. But try it out.. also you don't need to change the settings you don't want to override, so you could try:

```
observatory_model:
      camera:
        filter_mounted:
        - g
        - r
        - y
        filter_removable:
        filter_unmounted:
        - u
        - i
        - z
```

but again, I am not sure it works with less than 5 filters...

Reply:
Thanks, lets try